OrbitQt/orbitmainwindow.cpp

```diff
@@ -917,12 +917,15 @@ void OrbitMainWindow::closeEvent(QCloseEvent* event) {
   }
 }
 
-void OrbitMainWindow::SetupGrpcChannel(std::string grpc_server_address) {
+void OrbitMainWindow::SetupGrpcAndProcessManager(std::string grpc_server_address) {
   std::shared_ptr<grpc::Channel> grpc_channel = grpc::CreateCustomChannel(
       grpc_server_address, grpc::InsecureChannelCredentials(), grpc::ChannelArguments());
   if (!grpc_channel) {
     ERROR("Unable to create GRPC channel to %s", grpc_server_address);
   }
 
+  process_manager_ = ProcessManager::Create(grpc_channel, absl::Milliseconds(1000));
```

Comment:
Same comment as for the grpc_channel: Why the change in ownership?

Reply:
See comment above